release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -265,6 +265,24 @@ For more information, see xref:../nodes/clusters/nodes-cluster-cgroups-2.adoc#no
 
 With this release, you can increase the disk quota in etcd. This is a Technology Preview feature. For more information, see xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#etcd-increase-db_recommended-etcd-practices[Increasing the database size for etcd].
 
+[id="ocp-4-16-reserved-core-freq-tuning_{context}"]
+====  Reserved Core Frequency Tuning
```

Comment:
Can this be "Reserved core frequency tuning"? Jw if its necessary to capitalize all the words?

Reply:
yes that's what it should be. Thanks @skrthomas